apps/events/models/events.py

```diff
@@ -29,10 +28,8 @@ class Event(AbstractBaseModel):
         blank=False,
     )
     description = fields.TextField(max_length=250, null=True, blank=True)
-
-    location = PointField(default=Point(27.561831, 53.902284))
-    city_south_west_point = PointField(default=Point(27.561831, 53.902284))
-    city_north_east_point = PointField(default=Point(27.561831, 53.902284))
+    city_location = models.ForeignKey(City, on_delete=models.PROTECT, null=True)
+    place_id = fields.CharField(max_length=255, default="ChIJ02oeW9PP20YR2XC13VO4YQs", null=True, blank=True)
```

Comment:
place_id should be inside city location object

Reply:
Done